tools/release/check-changelog.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Expect as input parameter the commits range to analyze.
```

Comment:
For people not familiar with what a git revision is, it might be helpful to add a link to the docs in the comment: `https://git-scm.com/docs/gitrevisions`

Also maybe an example command in the comments might be nice, for instance:
```bash
$ ./tools/release/check-changelog.sh ORIG_HEAD..HEAD
```

Reply:
Sure, done.